openlibrary/coverstore/code.py

```diff
@@ -132,6 +133,7 @@ def error(code__msg):
             try:
                 data = download(source_url)
             except:
+                logger.exception("download({}) failed".format(source_url))
```

Comment:
Don't need this, since `error` will already trigger a sentry event. Also including the url in the error message will prevent sentry from being able to group these errors together. We'll get the url inside the sentry stack trace :+1:

```suggestion
```

Reply:
Will we see the original exception?  This is a bare exception so the current code sweeps exception details under the rug.